README.md

```diff
@@ -20,7 +20,7 @@ Visibility|All jobs|
 You can download the .NET SDK as either an installer (MSI, PKG) or a zip (zip, tar.gz). The .NET SDK contains both the .NET runtime and CLI tools.
 
 - [Official builds](https://dotnet.microsoft.com/download/dotnet)
-- [**.NET SDK latest builds table**](documentation/package-table.md)
+- For the latest SDK installers and installation instructions, see the [Installing the SDK section of the dotnet/dotnet repository](https://github.com/dotnet/dotnet#installing-the-sdk)
```

Comment:
I'd probably go straight to the build table as that's what this did before so just keep it simple like this:
[**.NET SDK latest builds table**](https://github.com/dotnet/dotnet/blob/main/docs/builds-table.md)